widgets/tabs/js/tabs.js

```diff
@@ -52,7 +52,7 @@ jQuery( function ( $ ) {
 				}
 			};
 			
-			$tabs.click( function () {
+			$tabs.on('keyup click', function () {
 				selectTab( this );
```

Comment:
This automatically selects the tab as soon as the Tab button is released, which isn't usual Tab button behavior. Pressing Tab should cycle focus to the tab and then it should only be selected when the Enter key is pressed.

Reply:
[It looks like both methods are accepted](https://www.w3.org/TR/wai-aria-practices/#tabpanel), but displaying the contents automatically upon focus is the general recommendation if there's no latency delay during the tab change (as in no requests to the server in general) ([source](https://www.w3.org/TR/wai-aria-practices/#kbd_selection_follows_focus))

Regardless, there are a few other recommendations (such as the tabindex and keyboard control) that should be made.

Reply:
Oh, that's interesting. In my general experience tab components use manual activation, at least, I can't recall seeing one use automatic activation. Anyway, all good then. 👍 

>Regardless, there are a few other recommendations (such as the tabindex and keyboard control) that should be made.

Not sure what you mean by this last bit? Recommendations? Was there something else you still wanted to change or add?

Reply:
The tabindex needs to only be set on the active tab and a tabindex must be set for the tab content.

You navigate through the tabs using arrows.

I would try [the example tab](https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html) for how the tab is ideally meant to work. Thoughts?

Reply:
Ah, ok. Yeah, I agree, let's go with the recommended functionality for better accessibility. 👍